src/event-helper.js

```diff
@@ -131,6 +131,7 @@ export function listenOncePromise(element, eventType, opt_evtListenerOpts,
  */
 export function isLoaded(eleOrWindow) {
   return !!(eleOrWindow.complete || eleOrWindow.readyState == 'complete'
+      || (eleOrWindow instanceof HTMLMediaElement && eleOrWindow.readyState > 0)
```

Comment:
`instanceof` will break across FIE boundaries. Use `typeof eleOrWindow.readyState === 'number'` instead.

Reply:
OMG, catch of the year. Took me 5 mins to even understand why. 👏 🙇 @jridgewell

Reply:
Thanks!! Done

Reply:
Oh, I probably should have explained. FIEs are different HTML documents, and have different `window`s, and that window has a difference `HTMLMediaElement` function instance. So `fie.querySelector('audio') instanceof fie.parent.HTMLMediaElement` is always false.